src/rdkafka.h

```diff
@@ -3849,12 +3897,23 @@ rd_kafka_consumer_group_metadata_new (const char *group_id);
  * @brief Create a new consumer group metadata object.
  *        This is typically only used for writing tests.
  *
+ * @param group_id The group id (must not be NULL).
+ *
```

Comment:
It is usually not needed to say that a pointer argument must not be NULL, that's the standard assumption, and as long as we say which ones are optional and can be NULL I think it is less ambigious in the overall picture if required fields are not explicitly called out as required, otherwise we would need to update all doc strings everywhere.

Reply:
sounds good